src/notification/test/container.test.ts

```diff
@@ -0,0 +1,39 @@
+import { elementUpdated, expect } from "@open-wc/testing";
+
+import { Notification } from "../index";
+
+describe("Notification", () => {
+  describe("container", () => {
+    it("should be append to body when not assign on constructor", async () => {
+      const container = new Notification();
+      container.open();
+      await elementUpdated(container);
+
+      const parrentEl = container.parentNode as HTMLElement;
+      expect(parrentEl.nodeName).to.equal("BODY");
+    });
+
+    it("should be append to body when assign invalid element on constructor", async () => {
+      const container = new Notification({
+        container: document.getElementById("some-id"),
```

Comment:
In this case container is null.
Please update it.

Reply:
I updated it